packages/cli/src/commands/doctor/healthchecks/__tests__/androidSDK.test.ts

```diff
@@ -0,0 +1,66 @@
+import execa from 'execa';
+import androidSDK from '../androidSDK';
+import getEnvironmentInfo from '../../../../tools/envinfo';
+import {EnvironmentInfo} from '../../types';
+import {NoopLoader} from '../../../../tools/loader';
+
+import * as common from '../common';
+
+const logSpy = jest.spyOn(common, 'logManualInstallation');
+
+const mockExeca = (stdout: string) => {
+  jest.mock('execa', () => jest.fn());
```

Comment:
`jest.mock` is hosited to the top of the parent scope, it's not meant to be called mutliple times. Can you move there so it's a bit more obvious the `execa` _module_ is mocked?

Reply:
Changed it a little bit to be easier to understand and more standard. What do you think?